examples/buildkit3/buildkit.go

```diff
@@ -109,6 +117,6 @@ func copyFrom(src *llb.State, srcPath, destPath string) llb.StateOption {
 func copy(src *llb.State, srcPath string, dest *llb.State, destPath string) *llb.State {
 	cpImage := llb.Image("docker.io/library/alpine:latest")
 	cp := cpImage.Run(llb.Shlexf("cp -a /src%s /dest%s", srcPath, destPath))
-	cp.AddMount("/src", src)
+	cp.AddMount("/src", src, llb.Readonly)
```

Comment:
Switching this was not related to the change. Just something I noticed while making other changes.